ui-components/src/navigation/ProgressNav.tsx

```diff
@@ -19,7 +19,9 @@ const ProgressNavItem = (props: {
 
   return (
     <li className={`progress-nav__item ${bgColor}`}>
-      <a href="#">{props.label}</a>
+      <a aria-disabled={bgColor == "is-disabled"} href={"#"}>
```

Comment:
Do you mind changing this to use strict equality (`===`)?

Reply:
Just updated.